src/cbl-mariner/2.0/fpm/amd64/Dockerfile

```diff
@@ -17,5 +17,7 @@ RUN tdnf install -y \
         # Provides functionality for AzureCLI AzDO task
         powershell \
         azure-cli \
+        # Provides functionality for MicroBuild install task
+        python3-pip \
```

Comment:
Should we explicitly add the `python3` package here?

While it is installed in the base image, it is a dependency of MicroBuild, so it may be good to list it here.

Reply:
We could. It gets installed with pip though in this case:

```
docker run --rm 4841e5ea63ba0dffda5636f3ce67cda96aa2b703a2d916e47a43e568d508787e python3 -V
Python 3.9.19
```

Reply:
Correction on my part, doesn't look like `python` comes in the base image for either CBL Mariner 2.0 or Azure Linux 3.0.